tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,120 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	assert.lacksClasses( element, "ui-state-disabled",
+		"ui-state-disabled is not present on widget after disabling" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 6,
+		"Child widgets are disabled" );
+});
+
+test( "enable", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "enable" );
+	assert.lacksClasses( element, "ui-state-disabled",
+		"ui-state-disabled is not present on widget after enabling" );
+	strictEqual( element.find( "ui-state-disabled" ).length, 0,
+		"Child widgets are disabled" );
+});
+
+function hasCornerClass( element, className ) {
+	if ( className ) {
+		return element.hasClass( className );
+	}
+
+	return !!element.attr( "class" ).match( /ui-corner/g );
+}
+test( "refresh", function() {
+	var count = 0,
+		tests = {
+			"checkboxradio": "<input type='checkbox'>",
+			"selectmenu": "<select><option>foo</option></select>",
+			"button": "<button>"
+		},
+		orientations = {
+			"horizontal": [
+				"ui-corner-left",
+				false,
+				false,
+				"ui-corner-right"
+			],
+			"vertical": [
+				"ui-corner-top",
+				false,
+				false,
+				"ui-corner-bottom"
+			]
+		};
+
+	$.each( tests, function( widget, html ) {
+		$.each( orientations, function( name, classes ) {
+			count += 41;
+			var i, control, currentClasses,
+				controls = [],
+				element = $( "<div>" ).controlgroup({
+					direction: name
+				}).appendTo( "body" );
+
+			for ( i = 0; i < 4; i++ ) {
+				control = $( html ).attr( "id", "id" + i )
+					.add( $( "<label>" ).clone().attr( "for", "id" + i ) );
+
+				controls.push( control );
+				element.append( control );
+			}
+			element.controlgroup( "refresh" );
+			for ( i = 0; i < 4; i++ ) {
+				strictEqual( controls[ i ].is( ":ui-" + widget ), true,
+					name + ": " + widget + " " + i + ": is a " + widget + " widget after refresh" );
+			}
+			checkCornerClasses( classes );
+			interateHidden( true );
+			element.controlgroup( "option", "excludeInvisible", false );
+			interateHidden();
+
+			controls[ 0 ].prop( "disabled", true );
+
+			element.controlgroup( "refresh" );
+
+			strictEqual( controls[ 0 ][ widget ]( "widget" ).hasClass( "ui-state-disabled" ), true,
+				"Checkbox has ui-state-disabled after adding disabled prop and refreshing control group" );
+
+			element.remove();
+			expect( count );
```

Comment:
Move to top of test scope and use number literal. The count doesn't actual change with each run of the test suite, right?